api/gateway.proto

```diff
@@ -155,7 +155,7 @@ message Gateway {
   google.protobuf.Duration schedule_anytime_delay = 19 [(gogoproto.stdduration) = true, (gogoproto.nullable) = true];
   // Update the location of this gateway from status messages. This only works for gateways connecting with authentication; gateways connected over UDP are not supported.
   bool update_location_from_status = 21;
-  // The LoRa Basics Station LNS secret.
+  // The LoRa Basics Station LNS secret, base64 encoded.
```

Comment:
This is incorrect. The `lbs_lns_secret` field is a `Secret` message, not base64.

_inside_ the `Secret` there's indeed a `bytes value` that, when using JSON, is encoded as base64. This shouldn't have to be documented in a proto comment (because it's already proto spec). 

Probably better to update the documentation template to mention base64 for any `bytes` field (there are some exceptions for fields with `gogoproto.customtype`, those we can indeed mention explicitly).

Reply:
> This is incorrect. The `lbs_lns_secret` field is a `Secret` message, not base64.
> 
> _inside_ the `Secret` there's indeed a `bytes value` that, when using JSON, is encoded as base64. This shouldn't have to be documented in a proto comment (because it's already proto spec).
> 
> Probably better to update the documentation template to mention base64 for any `bytes` field (there are some exceptions for fields with `gogoproto.customtype`, those we can indeed mention explicitly).

this won't solve this particular problem, because the definition of `lbs_lns_secret` at https://www.thethingsindustries.com/docs/reference/api/gateway/#message:Gateway has a broken link to type `Secret`. i think that this instance is not crucial, but we have to address it as part of a much needed API doc overhaul.